src/components/Card.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+function Card({
+  color, name, population, flag,
+}) {
+  return (
+    <div className={`homeCard ${color[0]}`}>
+      <div className="homeCardTop">
+        <i className="fa-regular fa-circle-right cardIcon" />
+        <img className="homeCardTopImg" src={flag} alt={name} />
+      </div>
+      <div className="homeCardBottom">
+        <p className="homeCardTitle">{name}</p>
+        <p className="homeCardNumber">
+          {Math.round(population / 1000000)}
+          M  People
+        </p>
+      </div>
+    </div>
+  );
```

Comment:
- [ ] Nice work on the style however you need to align the text on the right on the container and not in the middle as the requirements say that we need to follow the template design. check the difference bellow

the template | you design
---|---
<img width="306" alt="image" src="https://user-images.githubusercontent.com/65068771/236546767-02b85674-405b-465e-8875-73640342fee2.png"> | ![image](https://user-images.githubusercontent.com/65068771/236546932-ca866f4b-3578-4cae-bae3-d11cdb5e47c3.png)